src/WebARKitController.js

```diff
@@ -125,9 +125,13 @@ export default class WebARKitController {
       console.log('Height of image is: ', this.height)
 
       const canvas = createCanvas(this.width, this.height)
+      console.log('Creating the canvas...');
       const ctx = canvas.getContext('2d')
       ctx.drawImage(image, 0, 0)
       let data = ctx.getImageData(0, 0, this.width, this.height).data
+      console.log('we get the data...');
+
+      this.webarkit.imageSetup(this.width, this.height)
```

Comment:
that was causing the issue! i missed to call this function that allocate the pointer....
But now i get another error:

```
WebARKit  0.0.0
WebARKitController.js:70 [WebARKitController] WebARKit initialized
webarkit_ES6_wasm.js:4 Allocated videoSize: 1228800

webarkit_ES6_wasm.js:4 Allocated videoFrame, pointer is: 5524056

WebARKitController.js:72 [WebARKitController] Got ID from setup 0
webarkit_ES6_wasm.js:4 This is a test from WebARKitJS.cpp!
webarkit_ES6_wasm.js:4 Start to initialize tracking...
webarkit_ES6_wasm.js:4 Reference image not found. AR is unintialized!
WebARKitController.js:160 Width of image is:  1637
WebARKitController.js:161 Height of image is:  2048
WebARKitController.js:163 Creating the canvas...
WebARKitController.js:167 we get the data...
webarkit_ES6_wasm.js:4 Allocated image2DSize: 13410304

webarkit_ES6_wasm.js:4 Allocated image2DFrame, pointer is: 6752920

WebARKitController.js:177 Hey, i am here!
WebARKitController.js:178 1637
webarkit_ES6_wasm.js:4 Start to initialize tracker...
webarkit_ES6_wasm.js:4 Start!
webarkit_ES6_wasm.js:4 2000
webarkit_ES6_wasm.js:4 Orb created!
webarkit_ES6_wasm.js:4 BFMatcher created!
webarkit_ES6_wasm.js:4 segmentation fault
abort @ webarkit_ES6_wasm.js:4
segfault @ webarkit_ES6_wasm.js:4
(anonymous) @ 0072e8ee:0x19fbf2
(anonymous) @ 0072e8ee:0x32bc
(anonymous) @ 0072e8ee:0x30df
(anonymous) @ 0072e8ee:0x5532
(anonymous) @ 0072e8ee:0x581b
initTracking @ VM25224:10
eval @ WebARKitController.js:180
Promise.then (async)
loadTracker @ WebARKitController.js:157
(anonymous) @ example.html:14
Promise.then (async)
(anonymous) @ example.html:12
webarkit_ES6_wasm.js:4 Uncaught (in promise) RuntimeError: abort(segmentation fault) at Error
    at jsStackTrace (webpack://WebARKit/./build/webarkit_ES6_wasm.js?:4:2553977)
    at stackTrace (webpack://WebARKit/./build/webarkit_ES6_wasm.js?:4:2554161)
    at abort (webpack://WebARKit/./build/webarkit_ES6_wasm.js?:4:38096)
    at segfault (webpack://WebARKit/./build/webarkit_ES6_wasm.js?:4:25378)
    at <anonymous>:wasm-function[3714]:0x19fbf2
    at <anonymous>:wasm-function[51]:0x32bc
    at <anonymous>:wasm-function[45]:0x30df
    at <anonymous>:wasm-function[202]:0x5532
    at <anonymous>:wasm-function[216]:0x581b
    at WebARKit.initTracking (eval at new_ (webpack://WebARKit/./build/webarkit_ES6_wasm.js?), <anonymous>:10:10)
    at abort (webpack://WebARKit/./build/webarkit_ES6_wasm.js?:4:38127)
    at segfault (webpack://WebARKit/./build/webarkit_ES6_wasm.js?:4:25378)
    at <anonymous>:wasm-function[3714]:0x19fbf2
    at <anonymous>:wasm-function[51]:0x32bc
    at <anonymous>:wasm-function[45]:0x30df
    at <anonymous>:wasm-function[202]:0x5532
    at <anonymous>:wasm-function[216]:0x581b
    at WebARKit.initTracking (eval at new_ (webpack://WebARKit/./build/webarkit_ES6_wasm.js?), <anonymous>:10:10)
    at eval (webpack://WebARKit/./src/WebARKitController.js?:180:25)
abort @ webarkit_ES6_wasm.js:4
segfault @ webarkit_ES6_wasm.js:4
(anonymous) @ 0072e8ee:0x19fbf2
(anonymous) @ 0072e8ee:0x32bc
(anonymous) @ 0072e8ee:0x30df
(anonymous) @ 0072e8ee:0x5532
(anonymous) @ 0072e8ee:0x581b
initTracking @ VM25224:10
eval @ WebARKitController.js:180
Promise.then (async)
loadTracker @ WebARKitController.js:157
(anonymous) @ example.html:14
Promise.then (async)
(anonymous) @ example.html:12
```

maybe the C++ code i changed caused this one.